### ISSUE TITLE
Use "tags" instead of branches

### ISSUE THREAD

**Issue by robregonm** [fxpio/composer-asset-plugin#224]
I'm trying to import `"bower-asset/can-connect": "0.5.5"` (0.5.5 is a tag), however for some reason the tags are not "detected".
I tried other alternatives (like "asset-repositories" section, "dev-xxxxx", etc) without success.

Not sure if this is a bug or it is not supported yet.

Thanks in advance.

**Comment by francoispluchino** [comment]
Have you installed the plugin in global mode? See the [doc](https://github.com/francoispluchino/composer-asset-plugin/blob/master/Resources/doc/index.md).

Because for me:

```
$ composer install --prefer-dist -v
Loading composer repositories with package information
Updating dependencies (including require-dev)
Adding VCS repository bower-asset/can-connect
Reading bower.json of bower-asset/can-connect (v0.5.5)
Importing tag v0.5.5 (0.5.5.0)
Dependency resolution completed in 0.000 seconds
Analyzed 69 packages to resolve dependencies
Analyzed 69 rules to resolve dependencies
  - Installing bower-asset/can-connect (v0.5.5)
    Downloading: 100%
    Extracting archive

Writing lock file
Generating autoload files
```

**Comment by robregonm** [comment]
Yes, In fact, I've installed many packages before, however, for some reason I get this message:

```The requested package bower-asset/can-connect 0.5.5 exists as bower-asset/can-connect[dev-master] but these are rejected by your constraint.```

This is not the very first time that this happens, the way I use to fix is to remove the "vendor" directory and clean the composer cache, and then I reinstall all the composer packages.

Even, I installed the development version of the plugin, because I thought it could be related to #211 or #172 

However, no definitive solution so far.
Thanks in advance.
